rotkehlchen/chain/evm/nodes.py

```diff
@@ -1,27 +1,24 @@
-import json
-from pathlib import Path
 from typing import TYPE_CHECKING
 
-from rotkehlchen.fval import FVal
-
 if TYPE_CHECKING:
     from rotkehlchen.db.drivers.gevent import DBCursor
 
 
-def populate_rpc_nodes_in_database(write_cursor: 'DBCursor') -> None:
-    """Populates the rpc nodes in the datatabase from nodes.json
+def populate_rpc_nodes_in_database(
+        db_write_cursor: 'DBCursor',
+        globaldb_cursor: 'DBCursor',
+) -> None:
+    """Populates the rpc nodes in the user database from global database.
 
     This is supposed to run for a clean new DB
     """
-    dir_path = Path(__file__).resolve().parent.parent.parent
-    with open(dir_path / 'data' / 'nodes.json') as f:
-        nodes_info = json.loads(f.read())
+    # get rpc nodes from global db.
+    nodes = globaldb_cursor.execute(
+        'SELECT name, endpoint, owned, active, weight, blockchain FROM rpc_nodes',
+    ).fetchall()
```

Comment:
following this approach you are duplicating information and if you notice you have the same information both in the user's db and in the globaldb. Don't do this, just make the updates in the user's db